es/refresher-probability.md

```diff
@@ -1,347 +1,347 @@
 
-&#10230;
+&#10230; [Caso, Densidad marginal, Función acumulativa]
 
 <br>
 
 **39. Conditional density ― The conditional density of X with respect to Y, often noted fX|Y, is defined as follows:**
 
-&#10230;
+&#10230; Densidad condicional - La densidad condicional de X con respecto a Y, frecuentemente denotada como fX|Y, es definida como:
 
 <br>
 
-**40. Independence ― Two random variables X and Y are said to be independent if we have:**
+**40. Independence ― Two random variables X and Y are said to be independent if we have:** 
 
-&#10230;
+&#10230; Independencia - Dos variables aleatorias X y Y son consideradas independientes si se tiene:
 
 <br>
 
 **41. Covariance ― We define the covariance of two random variables X and Y, that we note σ2XY or more commonly Cov(X,Y), as follows:**
 
-&#10230;
+&#10230; Covarianza - Definimos la covarianza de dos variables aleatorias X y Y, denotada como σ2XY o comúnmente como Cov(X,Y), de la siguiente manera:
 
 <br>
 
 **42. Correlation ― By noting σX,σY the standard deviations of X and Y, we define the correlation between the random variables X and Y, noted ρXY, as follows:**
 
-&#10230;
+&#10230; Correlación - Sean σX,σY las desviaciones estándard de X y Y, definimos la correlación entre estas variables, denotada como ρXY, de la siguiente manera:
 
 <br>
 
 **43. Remark 1: we note that for any random variables X,Y, we have ρXY∈[−1,1].**
 
-&#10230;
+&#10230; Observación 1: Cabe resaltar que para X,Y variables aleatorias cualesquiera, se tiene que ρXY∈[−1,1].
 
 <br>
 
 **44. Remark 2: If X and Y are independent, then ρXY=0.**
 
-&#10230;
+&#10230; Observación 1: Si X y Y son independientes, entonces ρXY=0.
 
 <br>
 
 **45. Parameter estimation**
 
-&#10230;
+&#10230; Estimación de Parámetros
 
 <br>
 
 **46. Definitions**
 
-&#10230;
+&#10230; Definiciones
 
 <br>
 
 **47. Random sample ― A random sample is a collection of n random variables X1,...,Xn that are independent and identically distributed with X.**
 
-&#10230;
+&#10230; Muestra Aleatoria - Una muestra aleatoria es una colección de n variables aleatorias X1,...,Xn que son independientes e idénticamente distribuidas a X.
 
 <br>
 
 **48. Estimator ― An estimator is a function of the data that is used to infer the value of an unknown parameter in a statistical model.**
 
-&#10230;
+&#10230; Estimador - Un estimador es una función de los datos que es usada para inferir el valor de un parámetro desconocido en un modelo estadístico.
 
 <br>
 
 **49. Bias ― The bias of an estimator ^θ is defined as being the difference between the expected value of the distribution of ^θ and the true value, i.e.:**
 
-&#10230;
+&#10230; Sesgo - El sesgo de un estimador ^θ se define como la diferencia entre el valor esperado de la distribución de ^θ y el valor exacto, esto es:
 
 <br>
 
 **50. Remark: an estimator is said to be unbiased when we have E[^θ]=θ.**
 
-&#10230;
+&#10230; Observación: se dice que un estimador es insesgado cuando se tiene E[^θ]=θ.
```

Comment:
From a Mexican point of view, never heard insesgado, is commonly used just: "no sesgado"